.github/workflows/checks.yml

```diff
@@ -82,7 +82,7 @@ jobs:
       - name: Set up environment
         uses: ./.github/actions/setup
       - name: Run tests
-        run: forge test -vv
+        run: forge test -vv --ffi
```

Comment:
Npte: We agreed to not include that in the package. The foundry tests will be disabled in CI (until base64 eventually becomes foundray-native)